zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -232,15 +239,19 @@ async fn main() -> Result<()> {
 
                 let block_bytes: Vec<u8> = hex::decode(block_bytes)?;
 
-                // TODO: is it faster to call both `getblock height 0` and `getblock height 1`,
-                //       rather than deserializing the block and calculating its hash?
+                // TODO: is it faster to call both `getblock height version=0`
+                //       and `getblock height version=1`, rather than deserializing the block
```

Comment:
```suggestion
                // TODO: is it faster to call both `getblock height verbosity=0`
                //       and `getblock height verbosity=1`, rather than deserializing the block
```

Reply:
Oh, it's actually verbosity. Sorry, I missed it as well.